tools/wpt/revlist.py

```diff
@@ -0,0 +1,128 @@
+import argparse
+import calendar
+import logging
+import os
+import time
+from tools.wpt.testfiles import get_git_cmd
+
+here = os.path.dirname(__file__)
+wpt_root = os.path.abspath(os.path.join(here, os.pardir, os.pardir))
+
+logger = logging.getLogger()
+
+MYPY = False
+if MYPY:
+    # MYPY is set to True when run under Mypy.
+    from typing import Any
+    from typing import Dict
+    from typing import List
+    from typing import Text
+
+
+def calculate_min_timestamp(until, epoch, offset):
+    return ((((until - offset) // epoch)) * epoch) + offset
+
+
+def parse_epoch(value):
+    # type: (str) -> int
+    if isinstance(value, int):
+        return value
+    if value.lower() == "three-hourly":
+        return 10800
+    if value.lower() == "six-hourly":
+        return 21600
+    if value.lower() == "twelve-hourly":
+        return 43200
+    if value.lower() == "daily":
+        return 86400
+    if value.lower() == "weekly":
+        return 604800
+    return int(value)
+
+
+def get_tagged_revisions(pattern):
+    # type: (bytes) -> List[..., Dict]
+    '''
+    Returns the tagged revisions indexed by the committer date.
+    '''
+    git = get_git_cmd(wpt_root)
+    args = [
+        pattern,
+        '--sort=-committerdate',
+        '--format=%(refname:lstrip=2) %(objectname) %(committerdate:raw)',
+        '--count=100000'
+    ]
+    for tagged_revision in git("for-each-ref", *args).split("\n"):
+        revision_tag, revision_commit, revision_date, _ = tagged_revision.split(" ")  # noqa E501
+        revision_date = int(revision_date)
+        yield revision_tag, revision_commit, revision_date
+
+
+def list_tagged_revisons(epoch, max_count):
+    # type: (**Any) -> List[Text]
+    logger.debug("list_tagged_revisons(%s, %s)" % (epoch, max_count))
+    # Set an offset to start to count the the weekly epoch from
+    # Monday 00:00:00. This is particularly important for the weekly epoch
+    # because fix the start of the epoch to Monday. This offset is calculated
+    # from Thursday, 1 January 1970 0:00:00 to Monday, 29 December 1969 0:00:00
+    epoch_offset = -259200
+    # "epoch_threshold" set a safety margin after this time it is fine to
+    # consider that any tags are created and pushed.
+    epoch_threshold = 600
+    epoch_until = calendar.timegm(time.gmtime()) - epoch_threshold
```

Comment:
```suggestion
    epoch_until = int(time.time()) - epoch_threshold
```

Then you can also drop `import calendar`.